lib/streamlit/report_session.py

```diff
@@ -428,11 +428,11 @@ def handle_git_information_request(self):
                 msg.git_info_changed.state = GitInfo.GitStates.AHEAD_OF_REMOTE
             else:
                 msg.git_info_changed.state = GitInfo.GitStates.DEFAULT
+
```

Comment:
It looks like before, we would always send an empty ForwardMsg even if something failed in our Git import; now, we don't send that ForwardMsg.

Just wanted to check -- was this change on purpose, and is it necessary? Since the rest of the PR is just frontend code, ideally we don't have to change the Python code here; but eg if this is needed for our logic to be correct, or a test to pass here, I'm okay with it.

Reply:
It was failing a test due to this because we've been sending an empty forwardmsg

Reply:
Should I do a separate PR for this?